script.js

```diff
@@ -4,7 +4,7 @@ document.querySelector(".main-nav-toggle").addEventListener("click", () => {
 });
 
 // -------------------- FILTER SECTION  --------------------
-// Only runs filter code if it is on the right html file
+// Only runs filter code on the right html file
```

Comment:
While this is (generally) true and I realize you've only changed the comment here, not the actual logic, I would to point out that doing it this way has some major drawbacks.

For example, when I wanted to run your code I did so using a local HTTP server that serves on port 8000, not port 5500. Your code wouldn't run, and at first I couldn't understand why, until I found this.

A better way to do it would be to either add some code to the HTML file that initializes the JS code differently from each HTML file, or to check for something more consistent, like a class on the body element. Then you could have `class="challenges-page"` and `class="home-page"` respectively in the two HTML elements.